### DISCUSSION TITLE
Getting php segfault in symfony. How to debug?

### DISCUSSION THREAD

**Discussion by kriskoch** [symfony/symfony#45751] [Q&A] [answered]
I am getting a  php segfault in the latest version of php 8.1.3.
This is happening when I am trying to load a doctrine entity.
What is the easiest way to debug the segfault so I can file a bug report wherever I need to?

**Comment by lyrixx** [comment, marked as answer]
you can install xdebug, it might help (for infinite recursion for example)

Or I wrote an article to use only plain PHP to get some hint https://jolicode.com/blog/find-segfaults-in-php-like-a-boss

Or there is this french article: https://afsy.fr/avent/2017/05-deboguer-php-avec-gdb
